<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedSettingUi</class>
 <widget class="QWidget" name="AdvancedSettingUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">.QFrame {
border-radius: 10px;
background-color: #FFFFFF;
border: 1px solid #E0E0E0;
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>24</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>147</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>204</width>
            <height>149</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>204</width>
            <height>149</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../resource/resource.qrc">:/ui/images/img_ucar03.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,0,0">
        <property name="spacing">
         <number>30</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">QLabel {
font-family: SourceHanSansCN-Medium;
font-size: 18px;
color: #333333;
text-align: right;
}</string>
          </property>
          <property name="text">
           <string>系统：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_reset">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color: #007AFF;
border-radius: 4px;
font-family: SourceHanSansCN-Normal;
font-size: 14px;
color: #FFFFFF;
text-align: center;
}</string>
          </property>
          <property name="text">
           <string>恢复设置</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_guide">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color: #007AFF;
border-radius: 4px;
font-family: SourceHanSansCN-Normal;
font-size: 14px;
color: #FFFFFF;
text-align: center;
}</string>
          </property>
          <property name="text">
           <string>安装方向</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_checkOta">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color: #007AFF;
border-radius: 4px;
font-family: SourceHanSansCN-Normal;
font-size: 14px;
color: #FFFFFF;
text-align: center;
}</string>
          </property>
          <property name="text">
           <string>小车升级检查</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>177</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
